### PR TITLE
Feature/e UI 8379

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8379

### Change description ###
Document uploader in write query views
Refactor to avoid duplication

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-raise-query/query-write-raise-query.component.html [hunk 1]
```diff
@@ -1,150 +1,108 @@
...
+            <label class="govuk-label govuk-label--m govuk-!-font-weight-bold" for="query-raise-body">
+              {{ 'Query body' | rpxTranslate }}
+            </label>
+          </div>
+          <div id="query-raise-body-hint" class="govuk-hint">
+            {{ 'Include as many details as possible so case workers can respond to your query' | rpxTranslate }}
           </div>
+          <p id="query-raise-body-error" class="govuk-error-message"
+             *ngIf="formGroup.get('body')?.errors?.length">
+            <span class="govuk-visually-hidden">{{ 'Error:' | rpxTranslate }}</span> {{ 'Enter a body' |
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-raise-query/query-write-raise-query.component.html:68]: @MateiHMCTS - I think this error message should probably read `Enter a query body` as opposed to `Enter a body`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@MateiHMCTS - just one minor observation; the rest is OK

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by DavidJayakumar — APPROVED** [review]
Looks good

**Review by Daniel-Lam — DISMISSED** [review]

**Review by Daniel-Lam — APPROVED** [review]
@MateiHMCTS - this change with the `display_context` value looks fine; approved

**Pull request merged**